src/components/DocumentationTopic/TopicsLinkCardGridItem.vue

```diff
@@ -0,0 +1,93 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <Card
+    class="reference-card-grid-item"
+    :url="item.url"
+    :image="imageReferences.card"
+    :title="item.title"
+    floating-style
+    :size="compact ? undefined: 'large'"
```

Comment:
```suggestion
    :size="compact ? undefined : 'large'"
```

Super tiny nitpick, but this took me a minute to understand that this was a ternary expression without a space before the colon lol 😅

Also, do we have a constant for the `size` prop we could use instead of the string literal?